GameEngine/GameState.cs

```diff
@@ -50,9 +57,10 @@ public override bool Equals(object o)
             return other != null
                 && Board.Equals(other.Board)
                 && Deck.Equals(other.Deck)
-                && Hand.Equals(other.Hand)
+                && Hands.Count == other.Hands.Count && Hands.Zip( other.Hands, (a,b) => a.Equals(b) ).All( b => b )
```

Comment:
Can we have a line break before the second set of && on this line?

Reply:
👍